project/src/components/app/app.jsx

```diff
@@ -11,12 +11,17 @@ import offerProps from '../offers/offer.props';
 
 function App(props) {
   const { offers } = props;
+  const [currentPoint, setCurrentPoint] = useState({});
+
+  const getSelectedPoint = function(point) {
```

Comment:
```suggestion
  const handlePointSelect = function(point) {
```
Все локальные колбеки компонента названы через handle